spec/controllers/idv/phone_controller_spec.rb

```diff
@@ -225,7 +259,9 @@
           Analytics::IDV_PHONE_CONFIRMATION_VENDOR, result
         )
 
-        put :create, params: { idv_phone_form: { phone: '7035555555' } }
+        put :create, params: { idv_phone_form: { phone: bad_phone } }
+        expect(response).to redirect_to idv_phone_path
+        get :new
```

Comment:
what side effect does this trigger?

Reply:
The analytics are part of the behavior that runs after the async step is recognized as done: https://github.com/18F/identity-idp/blob/9adf76349af5b1b2703e0e08c020c6e042aca283/app/controllers/idv/phone_controller.rb#L94-L99

Reply:
Should we move the analytics expectation to after the `put` and before the `get` ?

Reply:
yep, moved

I was hoping to find a way to do those analytics outside of this controller, but it uses the result of the async call and the HTTP callback isn't in a session. I'm not sure on the best way to tie it all together outside of this 😕